doctr/datasets/vocabs.py

```diff
@@ -66,6 +69,7 @@
         + VOCABS["danish"]
         + VOCABS["finnish"]
         + VOCABS["swedish"]
+        + VOCABS["bangla"]
```

Comment:
Let's remove this ftm, because we don't have bangla data atm we could use to train a multilingual model which includes this vocab :+1: